rewrite-java/src/main/java/org/openrewrite/java/search/UsesType.java

```diff
@@ -25,21 +25,34 @@
 
 public class UsesType<P> extends JavaIsoVisitor<P> {
     private final Pattern typePattern;
+    private final Boolean includeImplicit;
 
-    public UsesType(String fullyQualifiedType) {
+    public UsesType(String fullyQualifiedType, Boolean includeImplicit) {
```

Comment:
Since this visitor is so very commonly used by recipes (as witnessed by the breaking builds for the three downstream projects we test this change against), I wonder if it in this case wouldn't make sense to overload the constructor and remain backward compatible. We could declare the single argument.constructor as `@Deprecated`. Thoughts?

Reply:
Thanks for clarifying. Let's move with this then and update `rewrite-spring` accordingly.

Reply:
sounds good; I'll probably merge this in an hour or two and start on corresponding changes for some of the downstream repos